Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -112,6 +152,43 @@ Common::Symbol* PPCSymbolDB::GetSymbolFromAddr(u32 addr)
   return nullptr;
 }
 
+Common::Note* PPCSymbolDB::GetNoteFromAddr(u32 addr)
+{
+  if (m_notes.empty())
+    return nullptr;
+
+  auto itn = m_notes.lower_bound(addr);
+
+  // If the address is exactly the start address of a symbol, we're done.
+  if (itn->second.address == addr)
```

Comment:
`itn` can be the end iterator here, you need to check that before dereferencing.